source/trees/sec-trees-in-action.ptx

```diff
@@ -13,12 +13,12 @@
             <ol>
                 <li>
                     <p>
-                        Find the latitude and longitude of each CCC campus location.
+                        Find the latitude and longitude of each CCC campus location. We are using latitude and longitude because we are calculating new railway paths that do not already exist on roads. Therefore, we cannot use addresses.
```

Comment:
The fact that the route does not exist does not preclude you from using addresses. I think the reason is more related to the fact that the formula you will use to find those distances calls for addresses given by latitude, longitude. I am not sure, but think about it. It is related to the fact that the route does not exist (that is why you are using the formula) but not directly.